distribution/rpm/vector.spec

```diff
@@ -76,8 +82,9 @@ rm -rf %{buildroot}
 %{_bindir}/*
 %{_unitdir}/vector.service
 %config(noreplace) %{_sysconfdir}/%{_name}/vector.toml
-%config(noreplace) %{_sysconfdir}/default/vector
 %config %{_sysconfdir}/%{_name}/examples/*
+%config %{_sysconfdir}/%{_name}/agent/*
+%config %{_sysconfdir}/%{_name}/aggregator/*
```

Comment:
So this seems angry if it's just copying over when it's just a `.gitkeep` file, I'd like to keep the directory structure for future defaults.

I could copy the `config/vector.toml` into it but the duplication feels a little weird.